### PR TITLE
ballin

### PR DESCRIPTION
**Pull request by ProtivogaSpriter**
added buyable chest rigs; just like in pre-rebase! (pouches coming soon)
also some resprites

### CODE REVIEW

**Comment by Marrone321** [comment]
Great job!

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -687,3 +687,27 @@
 /obj/item/storage/belt/sabre/PopulateContents()
 	new /obj/item/melee/sabre(src)
 	update_icon()
+
+/obj/item/storage/belt/military/tactibelt/ComponentInitialize()
+	. = ..()
+	GET_COMPONENT(STR, /datum/component/storage)
+	STR.max_items = 4
+	icon_state : "tactibelt"
```
Comment (ProbablyCarl) [inline code/game/objects/items/storage/belt.dm:695]: WHY NO JESUS FUCK WHY

Comment (ProtivogaSpriter) [reply code/game/objects/items/storage/belt.dm:695]: trolld epicly
:trollface:

Comment (ProtivogaSpriter) [reply code/game/objects/items/storage/belt.dm:695]: fr just change the : after the icon_state, item_state and desc with a = and it should work

Comment (Stalkeros2) [reply code/game/objects/items/storage/belt.dm:695]: What the FUCK is this shit.

**Pull request merged**